sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedManualValidationInfoAsync.java

```diff
@@ -0,0 +1,107 @@
+            .flatMap(trainingOperationResponse -> {
+                if (trainingOperationResponse.getStatus().isComplete()) {
+                    // training completed successfully, retrieving final result.
+                    return trainingOperationResponse.getFinalResult();
+                } else {
+                    return Mono.error(new RuntimeException("Polling completed unsuccessfully with status:"
+                        + trainingOperationResponse.getStatus()));
+                }
+            });
+
+        System.out.println("--------RECOGNIZING FORM --------");
+        recognizeFormResult.subscribe(recognizedForms -> recognizedForms.forEach(recognizedForm -> {
+            System.out.printf("Form has type: %s%n", recognizedForm.getFormType());
+            // each field is of type FormField
+            //     The value of the field can also be a FormField, or a list of FormFields
+            //     In our sample, it is not.
+            recognizedForm.getFields().forEach((fieldText, fieldValue) -> {
+                System.out.printf("Field %s has value %s based on %s within bounding box, with a confidence score "
+                        + "of %s.%n",
+                    fieldText, fieldValue.getFieldValue(), fieldValue.getValueText().getText(),
+                    fieldValue.getConfidence());
+            });
+
+            // Page Information
+            recognizedForm.getPages().forEach(formPage -> {
+                System.out.printf("-------Recognizing Page %s of Form -------%n", 1);
+                System.out.printf("Has width %s , angle %s, height %s %n", formPage.getWidth(),
+                    formPage.getTextAngle(), formPage.getHeight());
+                // Table information
+                System.out.println("Recognized Tables: ");
+                formPage.getTables().forEach(formTable -> {
+                    formTable.getCells().forEach(formTableCell -> {
+                        System.out.printf("Cell text %s has following words: %n", formTableCell.getText());
+                        // text_content only exists if you set include_text_content to True in your
+                        // function call to recognize_custom_forms
+                        // It is also a list of FormWords and FormLines, but in this example, we only deal with
+                        // FormWords
+                        formTableCell.getElements().forEach(formContent -> {
+                            if (formContent.getTextContentType().equals(TextContentType.WORD)) {
+                                FormWord formWordElement = (FormWord) (formContent);
+                                StringBuilder str = new StringBuilder();
+                                if (formWordElement.getBoundingBox() != null) {
+                                    formWordElement.getBoundingBox().getPoints().forEach(point -> {
+                                        str.append(String.format("[%s, %s]", point.getX(), point.getY()));
```

Comment:
```suggestion
                                        str.append(String.format("[%d, %d]", point.getX(), point.getY()));
```
there is other places that have using %s for numerical values.